### PR TITLE
Movie widgets

### PR DESCRIPTION
**Pull request by LeonardoMolleker**
ezgif com-gif-maker (10)

Ui to access to each movie widget.
Exported the widgets to use it in the main proyect.

### CODE REVIEW

File: lib/exports/movie_widgets.dart [hunk 1]
```diff
@@ -0,0 +1,10 @@
+library movie_widgets;
+export '../movie_widgets/fade_image_widget.dart';
+export '../movie_widgets/movie_detail_image.dart';
+export '../movie_widgets/movie_detail_play_button.dart';
+export '../movie_widgets/movie_details_actions.dart';
+export '../movie_widgets/movie_details_info.dart';
+export '../movie_widgets/movie_details_overview.dart';
+export '../movie_widgets/movie_details_title.dart';
+export '../movie_constants/movie_constants.dart';
+export '../movie_constants/movie_dimensions.dart';
```
Comment (MarcosCGlobant) [inline lib/exports/movie_widgets.dart]: missed the extra line at the end, do Ctrl + Alt + L to fix it, or add it manually

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

File: lib/exports/movie_widgets.dart (hunk 1 above)
Comment (LeonardoMolleker) [reply lib/exports/movie_widgets.dart]: Done.

**Review by FranciscoGaraventa — CHANGES_REQUESTED** [review]
In the cases of info_page, overview_page, play_button_page and title_page you're using the same structure on build the only difference that you have is the child inside Center on body change depending on what you are going to use, so try to create only one base page and pass with a parameter the widget that you re going to use.

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**